src/Avalonia.Input/IMouseDevice.cs

```diff
@@ -13,8 +13,10 @@ public interface IMouseDevice : IPointerDevice
         [Obsolete("Use PointerEventArgs.GetPosition")]
         PixelPoint Position { get; }
 
+        [Obsolete]
         void TopLevelClosed(IInputRoot root);
 
+        [Obsolete("Use IPointerDevice.SceneInvalidated")]
         void SceneInvalidated(IInputRoot root, Rect rect);
```

Comment:
Moved to IPointerDevice as it needs to be used in other pointer devices like PenDevice